src/containers/organization/OrganizationPageContainer.tsx

```diff
@@ -115,15 +115,23 @@ export const OrganizationPageContainer: FC<OrganizationPageContainerProps> = ({
   }, [usersWithRoles]);
 
   const contributions = useMemo(() => {
-    const { hubsHosting = [], challengesLeading = [] } = membershipData?.membershipOrganization || {};
+    const hubsHosting = membershipData?.rolesOrganization?.hubs?.filter(h => h.roles?.includes('host')) || [];
+    // const challengesLeading = membershipData?.rolesOrganization?.hubs?.flatMap(h => h.challenges).filter(c => c.roles?.includes('lead')) || [];
```

Comment:
I have left it commented here because at the end I was not using it, but thought it was good for future reference. I've just removed it, it's not needed